examples/04_New Models.ipynb

```diff
@@ -690,6 +690,140 @@
+    "        # 0 = x0 + v0*t - 0.5*g*t^2\n",
+    "        g = self.parameters['g']\n",
+    "        t_impact = -(x['v'] + np.sqrt(x['v']*x['v'] - 2*g*x['x']))/g\n",
+    "\n",
+    "        # 0 = v0 - g*t\n",
+    "        t_falling = -x['v']/g\n",
+    "                \n",
+    "        return {'falling': t_falling, 'impact': t_impact}\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With this, our direct model is created. Note that adding `*args` and `**kwargs` is optional. Having these arguments makes the function interchangeable with other models which may have arguments or keyword arguments. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's test out this capability. To do so, we'll use the `time` package to compare the direct model to our original timeseries model. \n",
+    "\n",
+    "Let's start by creating an instance of our timeseries model, calculating the time of event, and timing this computation. Note that for a state transition model, `time_of_event` still returns the time at which `threshold_met` returns true for each event, but this is calculated by simulating to threshold."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time \n",
+    "\n",
+    "m_timeseries = ThrownObject_ST()\n",
+    "x = m_timeseries.initialize()\n",
+    "print(m_timeseries.__class__.__name__, \"(Direct Model)\" if m_timeseries.is_direct else \"(Timeseries Model)\")\n",
+    "tic = time.perf_counter()\n",
+    "print('Time of event: ', m_timeseries.time_of_event(x, dt = 0.05))\n",
+    "toc = time.perf_counter()\n",
+    "print(f'execution: {(toc-tic)*1000:0.4f} milliseconds')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's do the same using our direct model implementation. In this case, when `time_to_event` is called, the event time will be estimated directly from the state, instead of through simulation to threshold. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m_direct = DirectThrownObject()\n",
+    "x = m_direct.initialize()  # Using Initial state\n",
+    "# Now instead of simulating to threshold, we can estimate it directly from the state, like so\n",
+    "print('\\n', m_direct.__class__.__name__, \"(Direct Model)\" if m_direct.is_direct else \"(Timeseries Model)\")\n",
+    "tic = time.perf_counter()\n",
+    "print('Time of event: ', m_direct.time_of_event(x))\n",
+    "toc = time.perf_counter()\n",
+    "print(f'execution: {(toc-tic)*1000:0.4f} milliseconds')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that execution is significantly faster for the direct model. This will be even more pronounced for events that occur later in the simulation. "
```

Comment:
```suggestion
    "Notice that execution is significantly faster for the direct model. Furthermore, the result is actually more accurate, since it's not limited by the timestep (see dt section in 01_Simulation). These observations will be even more pronounced for events that occur later in the simulation. "
```